docs/can-guides/contribute/including-new-modules.md

```diff
@@ -0,0 +1,111 @@
+- `CANJS_PARENT_TARGET`: is the name of the target area in CanJS. For example, if the module is a part of the "Ecosystem", then the value for `@parent` would be `can-ecosystem`. If the target is "Legacy" than the value is `can-legacy`.
+- `PATH_TO_PACKAGE.JSON` is a relative path from the main document file for the source repo. Considering the following structure...
+
+```
+- docs
+ - my-project.md
+- src
+- package.json
+```
+
+If `my-project.md` is the main document, then the relative path to the `package.json` is `../package.json`.
+
+This would result in a main document file that looked something like
+
+```md
+@ module {Object} my-project
+@ parent can-ecosystem
+@ package ../package.json
+
+> Documentation goes here
+```
+
+## Requiring source repo in CanJS repo
+
+To build the documentation, first, the source repo needs to be added to the CanJS project.
+
+```bash
+npm install <MODULE_NAME> --save
+```
+
+Once the project is added as a dependency, the source project's main file needs to be included in the CanJS project.
+
+In the root directory of the project, locate the parent target JavaScript file (`ecosystem.js`, `legacy.js`, etc). These files are located in the CanJS project root.
+
+```md
+- /canjs
+ - core.js
+ - ecosystem.js
+ - infrastructure.js
+ - legacy.js
+```
+
+Open the desired file and require the source project.
+
+```javascript
+require('<SOURCE_MODULE_NAME>');
+```
+
+## Require source tests in CanJS repo
+
+Open the test main file in the CanJS repo, located at `canjs/test/test.js`.
+
+Require the source repo's test main file in the appropriate area. For example, if source repo will go in "Legacy" area of the site, then add it to the "Legacy" section of the test file.
+
+## Add necessary markup to CanJS docs
+
+Open the main doc file, located at `./docs/can-canjs/canjs.md`.
+
+Again, add any necessary markup to the correct section (related to the target parent; legacy for legacy, etc).
+
+Follow this helpful markdown template:
+
+```md
+- **[<MODULE_NAME>]** <small><%<MODULE_NAME>.package.version%></small> <MODULE_DESCRIPTION>
+  - `npm install <MODULE_NAME> --save`
+  - <a class="github-button" href="<PROJECT_GITHUB_URL>" data-count-href="/<PROJECT_GITHUB_ORG>/<MODULE_NAME>/stargazers" data-count-api="/repos/<PROJECT_GITHUB_ORG>/<MODULE_NAME>#stargazers_count">Star</a>
+```
+
+- `MODULE_NAME`: The project name, which should match the `name` property in the source project's `package.json`.
+- `MODULE_DESCRIPTION`: short description of project, can match the `description` property in source project's `package.json`.
+- `PROJECT_GITHUB_URL`: The Github url. Not the `git` path but the path to the html site.
+- `PROJECT_GITHUB_ORG`: The organization id or user id that owns the project in Github.
+
+
+## Notes
+
+The last part of the process is to test the documentation and test process locally. Once the site build is verified, create a PR against the CanJS repo to submit changes.
```

Comment:
This wording is a little hard to parse: “to test the documentation and test process locally”

Suggestion: “to build the documentation and run the tests locally”

Also, consider “in” instead of “against”.